ebos/eclgenericcpgridvanguard.cc

```diff
@@ -125,9 +126,12 @@ doLoadBalance_(const Dune::EdgeWeightMethod            edgeWeightsMethod,
         auto loadBalancerSet = static_cast<int>(externalLoadBalancer.has_value());
         this->grid_->comm().broadcast(&loadBalancerSet, 1, 0);
 
-        const auto faceTrans = ((loadBalancerSet == 0) || partitionJacobiBlocks)
-            ? this->extractFaceTrans(gridView)
-            : std::vector<double>{};
+        std::vector<double> faceTrans;
+        {
+            OPM_TIMEBLOCK(extractTrans);
+            if (loadBalancerSet == 0 || partitionJacobiBlocks)
+                this->extractFaceTrans(gridView);
```

Comment:
Don't you need to assign the return value to `faceTrans`?

Reply:
yes! nice catch (rebasing meh).